tango/step_caches/remote_step_cache.py

```diff
@@ -155,3 +155,11 @@ def __setitem__(self, step: Step, value: Any) -> None:
 
         # Finally, add to in-memory caches.
         self._add_to_cache(step.unique_id, value)
+
+    def __delitem__(self, step_unique_id) -> None:
+        # check and delete local cache dir
+        if self.step_dir(step_unique_id).is_dir():
```

Comment:
We can use `super().__delitem__()` instead of replicating what we do in local cache. Keeps the local and remote logic separate. If we change local's implementation, we won't need to change remote's implementation.